src/test/models/AuthPage.ts

```diff
@@ -123,7 +123,7 @@ export class AuthPage {
     return this.locator
       .page()
       .route(`**/self-service/${flow}/browser**`, async (route) => {
-        route.fulfill({
+        await route.fulfill({
```

Comment:
Do we not have eslint set up in this repo? Or did playwright change the types?

Reply:
i think we do have eslint but not for playwright - i'll add it quick. 

This seemed racy after I updated playwright, their docs also show this needs to be awaited.

Reply:
Generally, everything that returns a promise should always be awaited (except for very rare cases).

And especially in playwright missing `await`s proved very annoying to debug.

Reply:
yeah not sure why this wasn't awaited before :man_shrugging:. Somehow missed it, but the tests still passed xD